ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -423,7 +426,7 @@ def sac_entropy_loss(
                 )
             # We update all the _cont_ent_coef as one block
             entropy_loss += -1 * ModelUtils.masked_mean(
-                _cont_ent_coef * target_current_diff, loss_masks
+                torch.mean(_cont_ent_coef) * target_current_diff, loss_masks
```

Comment:
Bad merge? Looks like this is undoing https://github.com/Unity-Technologies/ml-agents/pull/4725